packages/core/src/events/DefaultEventHandlers.ts

```diff
@@ -150,6 +150,8 @@ export class DefaultEventHandlers<O = {}> extends CoreEventHandlers<
               return;
             }
 
+            store.actions.setNodeEvent('draggedOver', targetId);
+
```

Comment:
Could we possibly use `hovered` instead of having to add a `draggedOver` property to the state?

Then in the `getDraggedOverNodes` helper, we could do something like:
```
const hovered = state.events.hovered;
const dragged = state.events.dragged;

// If there's no nodes being dragged -
// then any hovered nodes are just being "regularly" hovered
if (dragged.size() === 0 || hovered.size() === 0) {
   return new Set();
}

// Might want to filter out the dragged node from the hovered set
return hovered;
```

Reply:
I finally took up the PR again and I really like your suggestion. There are a few problems though :'(

Currently `hovered` only updates on `mouseover`. For this suggestion to work, I would need to also add `store.actions.setNodeEvent('hovered', targetId);` to `dragenter` since `mouseover` is not triggered when dragging.

One could consider this a breaking change because now `hover` would be true when dragging nodes and current users would need to update their `useNode`s to only show their hovered state when we're not dragging.

But the biggest problem would be that `state.events.dragged` does nodes from `connectors.create`. And sadly cannot add `actions.setEvent('dragged', tree.rootNodeId)` to the `dragstart` of `connectors.create` because the tree has not yet been added.

---

To recap: We could leverage `hovered` and `dragged` when we A) add a breaking change regarding hovered and B) solve the problem of finding out whether a node is being created with `connectors.create` right now.

Reply:
Okay. I found a solution for the second problem. We can use the `indicator` instead of `dragged`

```
    getDraggedOverNodes(): Set<NodeId> {
      const hoveredNodes = state.events.hovered;

      if (hoveredNodes.size === 0 || state.indicator === null) {
        return new Set();
      } else {
        const hoveredId = Array.from(hoveredNodes)[0];

        return new Set([
          hoveredId,
          ...NodeHelpers(state, hoveredId).ancestors(),
        ]);
      }
    },
```

Reply:
Hm I see, let's maybe try it out and see how it works with the landing example (because there we make use of the `hovered` state to display the highlighted borders).

Otherwise, yeah maybe a new event `draggedOver` might be the way to go; still gotta warm up to that name though.